### PR TITLE
Setup Application Database and Tables

### LINKED ISSUE #4: [4]Setup the application database and tables
Setup and configure the application database and tables. Make sure you write all required tests.

### PR DESCRIPTION
**Pull request by iTaylorCodes**
### What does this PR do?
- This Pull Request sets up the application database and tables with tests for the models and methods.
- This PR closes #4 
### Description of Task to be completed
- Create the database and testing database using PostgreSQL. Setup User, Match, and Favorites, models using SQLAlchemy. Add class methods on User model for Signup and Authenticate. Add match methods to match 2 users together. Create tests for all models and methods.
### How should this be manually tested?
- First use "export FLASK_ENV=testing" to enter the testing environment. Then use "python -m unittest discover tests".

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -5,12 +5,20 @@ class Config(object):
     TESTING = False
 
     SECRET_KEY = secrets.token_hex(16)
+    SQLALCHEMY_ECHO = False
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    DEBUG_TB_INTERCEPT_REDIRECTS = False
+    SQLALCHEMY_DATABASE_URI = 'postgresql:///gamr'
```
Comment (alexkayabula) [inline config.py:11]: Add a parent configuration class from which these settings can be inherited by other configuration classes.

**Review by alexkayabula — CHANGES_REQUESTED** [review]
@iTaylorCodes set up the **travis.yml** file to build and run tests from the **test database**.

**Comment by alexkayabula** [comment]
@iTaylorCodes change the title of this PR. Remove `with Tests`.

**Comment by iTaylorCodes** [comment]
@alexkayabula Added PostgreSQL database to travis.yml. Tests are running through Travis CI now 👍

**Review by alexkayabula — APPROVED** [review]
LGTM

**Pull request merged**